### DISCUSSION TITLE
No V1 and V2? Here's where to find 12V

### DISCUSSION THREAD

**Discussion by mbo18** [omerfaruk-aran/esphome_samsung_hvac_bus#39] [Show and tell]
Hi!

I have a Windfree Samsung HVAC from 2020 with F1 and F2 next to GND and 230V but no V1 or V2. I was like "Nooooo" but find out that there's a specific connector (called Sub PBA in the doc) for an external remote. On this kind of HVAC, there's a connector for a Sub PCB additional card (ref MIM-A00N) where the remote is then connected. On this port 12V and GND are on PIN 9 (white = GND) and 10 (red = 12V)

![IMG_9423](https://github.com/lanwin/esphome_samsung_ac/assets/12999535/a2577932-ecce-4eec-8be5-c591ec73616d)

**Comment by lanwin** [comment]
Well what is your question? 

If you want to use 12v you need a step down connector. My experience is that 5v-9v works with the D1 Mini. 

BTW you dont need to put the device within the indoor unit. You also could put it into the outdoor unit or in between. At some place where you have a dedicated power connection available. Or an alternative would be to use the 220v lines and by a small power supply and put it in there.

**Reply by mbo18** [reply]
Well no question :) 
I wrote this in a "Show and tell" discussion just to help other if they don’t find V1 and V2 inside the indoor unit. 
I use a m5stack ESP32 exactly like in the doc and I do not have a power socket next to the unit

**Comment by atanasenko** [comment]
@mbo18 thank you for this hint. Got the m5 combo yesterday, connected, flashed and ran it today.
@lanwin thank you for this great project, my home just got smarter! How can I buy you a coffee? :D
![image](https://github.com/lanwin/esphome_samsung_ac/assets/266280/9d288e6d-e058-474e-ac14-b27ee72ac592)

**Reply by lanwin** [reply]
@atanasenko If you really want the easiest way would be to use my paypal page https://www.paypal.com/paypalme/lanwin . Thank you!

**Comment by kst84** [comment]
@mbo18 thanks a lot! It works like a charm, I don't need to use 230-12 power adapter anymore! 
![441986647_1136241337695721_5283461993170034166_n](https://github.com/lanwin/esphome_samsung_ac/assets/23737762/868d7c52-f878-4279-9f66-546816581f16)

**Reply by lanwin** [reply]
I will add this post to the readme

**Comment by hnykda** [comment]
Thanks @mbo18 ! I managed to get 5V from that 12V (also one 12v->3.3v, much smaller) with a step down buck converter as I use just ESP32.
![image](https://github.com/lanwin/esphome_samsung_ac/assets/2741256/2c32a754-d63e-4da8-9afc-9f6c437913b4)

Note that my cable pin 9 wasn't red, but orange.

**Comment by wifi75** [comment]
hello I have no Nasa samsung 2015 from my tester have found +12 voltage but after connect the module, the voltage drops to about 8Volts, why?

**Reply by wifi75** [reply]
![photo_5936058537727870280_y](https://github.com/user-attachments/assets/34a6cd2e-d133-4eb4-8793-4140810b0455)
![photo_5936058537727870282_y](https://github.com/user-attachments/assets/4931f5fe-8b6d-4206-86b0-02e2b45d2e74)
![photo_5936058537727870283_y](https://github.com/user-attachments/assets/59868863-e2cf-4ba1-a4f9-4552aebee031)

**Reply by hnykda** [reply]
I noticed the same thing - I was able to measure 12V on some pins, but as soon as I connected my ESP32, it dropped to 4V. I think you just cannot use that, even if the voltage is correct, the current is not strong enough. Try different pins.

**Reply by wifi75** [reply]
this solution?

https://www.amazon.it/Trasformatore-Trasformatori-Tensione-Costante-Striscia/dp/B0C9PCZTT1/ref=sr_1_8?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=UADR5NBK8EB&dib=eyJ2IjoiMSJ9.za0vHSaKaZxveQoRtRHzu5-Kv9mi-P-f9VihE2CSMOkzSH8mBaqZAcPUvLL7D62MCsMoShhKHb-03TXBSNgC3n2BwT_0cvaVTw16WSEEqfQSt4ERmiV2BYS3J7U__xCA3ccwL-P2eKu1KzqdEgwNL3M4pwARMd_EXJmfVNWITnxpefPobI9zxdr-1x8FVC40bwyHJGl-0CeiruvWHTAlE3RvcKwc9F7-Ji-WJLOA8JE.A10gygHDNco1lGE6vbIJt_ouMBu-EEE1bTHkohw4HcI&dib_tag=se&keywords=220v+12v&qid=1724604988&s=tools&sprefix=220+12v%2Cdiy%2C123&sr=1-8

**Reply by hnykda** [reply]
That should work for drawing the power from the mains (which are available on the similar place where this communication cable you are trying right now, at least on ACs I have seen). But my guess is that you will have some other 12V DC usable connection available at the cable you are trying now, making this transformer unnecessary. At least I would try that first before trying to get it from 230V through a transformer. 

It seems you found the ground. So then try every other pin for DC+ to see if you get stable 12V from that?

**Reply by mani94321** [reply]
> hello I have no Nasa samsung 2015 from my tester have found +12 voltage but after connect the module, the voltage drops to about 8Volts, why?

were you able to get it worked with non NASA AC?

**Comment by mani94321** [comment]
hi, i have Samsung AR12TSHZGWKY air condition, in the installation manual it mentions it dont have F1 F2. Is it possible to connect it with ESP with this project? Appreciate your support here.
![Screenshot 2024-09-26 132834](https://github.com/user-attachments/assets/e01dc1e0-3d8f-4d7a-a0c3-f0fea0205346)

**Reply by lanwin** [reply]
As far as I know, its not possible then. Sorry.

**Comment by zeratul021** [comment]
Can confirm the original post for indoor models AR12TXEAAWKN and AR09TXEAAWKN. Both have the same cable and pins 9 and 10 deliver enough power for the Atom.

**Comment by Lamieur** [comment]
Same for my AR60F**C1AWN (you know, modern "AI" models, imagine connecting your AC to the cloud ;)), I see 12,5V on these pins. Cheers mbo18!
